go/cmd/pfsserver/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"flag"
```

Comment:
Please use "github.com/namsral/flag" (a drop in replacement of "flag") instead of "flag", since the former can parse env variable as input.
Please see this discussion: https://github.com/PaddlePaddle/Paddle/pull/2245#discussion_r118412580

Usage of "github.com/namsral/flag" see: https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/go/cmd/pserver/pserver.go#L9